frontend/src/metabase/dashboard/components/RefreshWidget/RefreshWidgetTarget.tsx

```diff
@@ -0,0 +1,50 @@
+import { t } from "ttag";
+
+import { CountdownIcon } from "metabase/components/icons/CountdownIcon";
+import { DashboardHeaderButton } from "metabase/dashboard/components/DashboardHeader/DashboardHeader.styled";
+import { isNotNull } from "metabase/lib/types";
```

Comment:
(not related to this PR, but I though `lib/types` is about types, I'm surprised to see functions here)

Reply:
😄